plugins/com.google.cloud.tools.eclipse.appengine.deploy/plugin.xml

```diff
@@ -43,4 +43,28 @@
     </handler>
   </extension>
 
+  <extension
+    point="org.eclipse.ui.console.consolePatternMatchListeners">
+    <consolePatternMatchListener
+      class="com.google.cloud.tools.eclipse.ui.util.console.PatternToHyperlinkConverter"
+      id="com.google.cloud.tools.eclipse.appengine.deploy.consolePatternMatchListener"
+      regex="https://.+\.appspot\.com">
+      <enablement>
+        <test
+          property="com.google.cloud.tools.eclipse.ui.util.console.name"
+          value="App Engine Deploy">
```

Comment:
I agree. This comes from the fact that the `MessageConsole` for deploy comes from the `MessageConsoleUtilities` getConsole() method which uses the name.
For now I'd leave this as is in this PR, file an issue to use type to obtain the console and then use type in the property tester and fix it ASAP.